src/Momento.Sdk/Config/Transport/StaticTransportStrategy.cs

```diff
@@ -16,9 +17,15 @@ public class StaticGrpcConfiguration : IGrpcConfiguration
     /// <inheritdoc/>
     public int MinNumGrpcChannels { get; }
     /// <inheritdoc/>
-    public GrpcChannelOptions GrpcChannelOptions { get; }
+    public GrpcChannelOptions? GrpcChannelOptions { get; }
```

Comment:
This line triggered warnings in GitHub actions. Would you look at that? Otherwise looks good to go 👍